rollup.config.js

```diff
@@ -3,17 +3,18 @@ import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from 'rollup-plugin-typescript2';
 import postcss from 'rollup-plugin-postcss';
+import json from '@rollup/plugin-json';
 
 const packageJson = require('./package.json');
 
 export default {
 	input: packageJson.exports.require,
 	output: [
 		{
-			file: packageJson.exports.default,
+			dir: packageJson.exports.default,
```

Comment:
Why did you change this?

Reply:
it was in response to an error with this message `Error: When building multiple chunks, the output.dir option must be used, not output.file.`     Output.dir is used when multiple javascript chunks are generated  in  preference to output.file when just one javascript chunk is generated. https://rollupjs.org/guide/en/#outputdir

Reply:
This seems suspicious. The structure of the package shouldn't have changed. Maybe this is related to the deleted build artifacts and new incorrectly located file `build/index.js/index.js`.

Reply:
good point, but I updated the file location and reverted to using output.file and it produced the same error `[!] Error: Invalid value for option "output.file" - when building multiple chunks, the "output.dir" option must be used, not "output.file". To inline dynamic imports, set the "inlineDynamicImports" option.`